scripts/get-stalkerware-indicators.py

```diff
@@ -0,0 +1,64 @@
+import os
+import yaml
+import csv
+import sys
+
+# set current path to root
+os.chdir(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# setting path
+sys.path.append(os.getcwd())
+
+import config
+
+def requirements():
+    # check if submodule is initialized
+    if not os.path.exists(config.IOC_PATH) and not os.path.exists(config.IOC_FILE):
+        print("Submodule not initialized properly")
+        return False
+
+    # check config.APP_FLAGS_FILE exists
+    if not os.path.exists(config.APP_FLAGS_FILE):
+        print("app-flags.csv not found")
+        return False
+    return True
+
+# check requirements
+if not requirements():
+    print("Requirements not met. Exiting...")
+    exit()
+
+# parse ioc.yaml
+ioc = {}
+with open(config.IOC_FILE, "r") as f:
+    ioc = yaml.load(f, Loader=yaml.FullLoader)
+
+# get packages from every element of ioc dict
+apps = [element['packages'] for element in ioc if 'packages' in element]
+
+# print all indicators
+print("Found " + str(len(apps)) + " apps from the IOC stalkware indicators repostiory!")
+
+# read app-flags.csv csv file
+old_apps = []
+with open(config.APP_FLAGS_FILE, "r") as f:
+    reader = csv.reader(f)
+    for row in reader:
```

Comment:
again, no `try`/`except` here? `csv.reader` can throw `csv.Error`

Reply:
oh, I added a method to check the prereqs in the `requirements()` method

Reply:
That function does not ensure that `APP_FLAGS_FILE` is valid CSV nor that `IOC_FILE` is valid YAML. These functions can throw exceptions if the underlying files are 1) missing, which you check for; 2) invalidly formatted, which nothing here checks for; 3) contain utf-8 symbols (I believe by default these libraries expect ascii -- that may have changed in Python3?) etc.

The attempts to read and write should be wrapped. Two examples:

[reading yaml](https://github.com/stopipv/isdi/blob/588942cece7b49e1f341fc399ccc24cd8518612a/scripts/get-stalkerware-indicators.py#L33):
```py
ioc = {}
try:
    with open(config.IOC_FILE, "r") as f:
        ioc = yaml.load(f, Loader=yaml.FullLoader)
except yaml.YAMLError as e:
    print("Error parsing YAML IOC file: {}".format(e))
    sys.exit(1)
except Exception as e:
    print("Other error reading IOC file: {}".format(e))
    sys.exit(1)
```

[reading csv](https://github.com/stopipv/isdi/blob/588942cece7b49e1f341fc399ccc24cd8518612a/scripts/get-stalkerware-indicators.py#L45)
```py
try:
    with open(config.APP_FLAGS_FILE, "r") as f:
        reader = csv.reader(f)
        old_apps = [row[0] for row in csv.reader(f)]
except csv.Error as e:
    print("Error parsing CSV app flags file: {}".format(e))
    sys.exit(1)
except Exception as e:
    print("Other error reading app flags file: {}".format(e))
    sys.exit(1)
```
```